### ISSUE TITLE
Better UI integration with FlightRadar24 in addition to FlightAware

### ISSUE THREAD

**Issue by e2jk** [Thom-x/docker-fr24feed-piaware-dump1090#44]
## Specifications

  - Image hash or version: [latest/1.3.1](https://hub.docker.com/layers/thomx/fr24feed-piaware/1.3.1/images/sha256-4e09b38fad0db1079e387eccddf4414fc5fc67eb21996ec482be23b02f438ca1?context=explore), contains [v3.8.1 of flightaware/dump1090](https://github.com/flightaware/dump1090/releases/tag/v3.8.1).
  - Platform: Raspberry Pi 1B
  - Subsystem: armv6

## Expected Behavior
As this Docker image combines both PiAware and FR24feed, I'd expect the UI to provide not just links to FlightAware but also to FlightRadar24, and a link to the local FR24Feed page, typically http://localhost:8754/
And why not, also a link to the public pages of one's own user/tracker on both platforms so that you can easily check the health of your radar, as seen externally by those 2 tracking sites:
- https://flightaware.com/adsb/stats/user/e2jk
- https://www.flightradar24.com/account/feed-stats/?id=34513

## Actual Behavior
Data upload is performed to both platforms, which is great.
But the UI elements are only FlightAware-focused
- Logos (which I don't really care about), FlightAware and PiAware/SkyAware
- Page title ("FlightAware SkyView")
- Links to the flights pages, both clicking on the flight number link or the "Visit Flight Page" button when a flight is selected:

![Capture d’écran de 2021-02-28 10-53-14](https://user-images.githubusercontent.com/229379/109414589-f2876b00-79b3-11eb-87fa-a335d2ff9da1.png)

I don't expect FlightAware to modify their version of the dump1090 package to include links to one of their competitors, though ;)
What would be the most efficient way to get these changes, would we need to fork after each stable release and add these items/ Probably better than to try to monkey-patch directly from the Dockerfile...
Any interest to work (together) on this effort? I could create a new repo that downloads the latest stable release of flightaware/dump1090 and modifies/adds these elements, would that be something you'd be interested in?

The links to the user/tracker page on both platforms could be passed on as environment variables when launching the Docker container, like is done for all the other parameters of this container.

**Note:** I am not professionally affiliated to any of the 2 platforms in any way, just a simple user.
I have been a FR24 user for a long time (started my first receiver in 2013, pushing only to FR24) and just recently created my FlightAware profile and pushing to FA thanks to this repo. If it hadn't been for the easy integration with the 2 platforms that is facilitated by your Docker container @Thom-x, I would likely only have continued to push data to FR24 and not FlightAware.
Just adding this in case my suggestion here would be misinterpreted ("all this hard maintenance work done by FlightAware, and this guy just wants to tag along FR24"), it is thanks to this combination that I'm now also feeding FlightAware...

## Steps to Reproduce the Problem

Regular install of the Docker image.

**Comment by Thom-x** [comment]
It could be a good idea ! I don't really have time to do that but I would be happy to accept PR for it (and review them).

I'm not a front end expert, but I could help on the Docker part. 👍

**Comment by e2jk** [comment]
Yes, time is an issue ;) I'm putting this on my todo list.

**Comment by Thom-x** [comment]
Thank you !

**Comment by e2jk** [comment]
Hi @Thom-x , I've set up an initial patching setup that adds the FR24 logo and uses a neutral page name. This is how it looks like until now:

![Capture d’écran de 2021-02-28 20-09-37](https://user-images.githubusercontent.com/229379/109430322-0907e380-7a01-11eb-8112-878cba93582f.png)

Before I go too far down that road, could you give it a try and tell me if that is in line with what you'd expect?

I've added a bit of explanation here: https://github.com/e2jk/docker-fr24feed-piaware-dump1090/tree/issue-44/patch
Basically I created a .patch file that modifies 4 files from the upstream release, and that file gets plugged into the Dockerfile.

Note: The patch works fine for v3.8.1 (the current target of "latest"), but fails on v4.0 because upstream added extra whitespace in the `public_html/script.js` file, in the `updatePiAwareOrFlightFeeder()` function that gets patched. Simple to update when v4.0 (or likely 4.1) is ready for usage.

**Comment by e2jk** [comment]
Just to add a bit more context:
- The "real" work happens [here](https://github.com/e2jk/docker-fr24feed-piaware-dump1090/commit/5bcd6aefbb175914233a4ad8dc22c3e068b013ff) in the Dockerfile, in which:
  - The FR24 logo gets copied 
  - The [patch/flightradar24.patch](https://github.com/e2jk/docker-fr24feed-piaware-dump1090/blob/issue-44/patch/flightradar24.patch) file gets applied that modifies the index page to add the logo and the js files for a neutral page name

**Comment by Thom-x** [comment]
Great ! You can do a PR and we will create a new docker tag to test if it's OK.

**Comment by e2jk** [comment]
PR created with the code until now. I'm running a Raspi 1B, when creating/publishing the new Docker tag can you make sure to upload also the arm version? Thanks.

**Comment by e2jk** [comment]
Thanks for meeting #45. Since there are more changes coming up in #49, @Thom-x can you reopen this issue?

**Comment by e2jk** [comment]
Great, thanks for merging both PRs!
